docs/pages/getting-started/adapters/drizzle.mdx

```diff
@@ -196,7 +198,9 @@ import type { AdapterAccount } from "next-auth/adapters"
 import { randomUUID } from "crypto"
 
 export const users = mysqlTable("user", {
-  id: varchar("id", { length: 255 }).primaryKey().$defaultFn(() => randomUUID()),
+  id: varchar("id", { length: 255 })
+    .primaryKey()
+    .$defaultFn(() => randomUUID()),
```

Comment:
docs should show usage using global crypto too to prevent confusion regarding runtime compatibility